clients/java/signalr/build.gradle

```diff
@@ -19,6 +19,7 @@ dependencies {
     implementation "org.java-websocket:Java-WebSocket:1.3.8"
     implementation 'com.google.code.gson:gson:2.8.5'
     implementation 'org.apache.httpcomponents:httpclient:4.5.6'
+    implementation 'com.squareup.okhttp3:okhttp:3.11.0'
```

Comment:
We need to do an OSS request 

cc @Eilon

Reply:
Already sent an email before updating the PR